lib/experiments-page.php

```diff
@@ -124,6 +135,7 @@ function gutenberg_experiments_editor_settings( $settings ) {
 		'__experimentalEnableLegacyWidgetBlock' => $experiments_exist ? array_key_exists( 'gutenberg-widget-experiments', get_option( 'gutenberg-experiments' ) ) : false,
 		'__experimentalEnableMenuBlock'         => $experiments_exist ? array_key_exists( 'gutenberg-menu-block', get_option( 'gutenberg-experiments' ) ) : false,
 		'__experimentalBlockDirectory'          => $experiments_exist ? array_key_exists( 'gutenberg-block-directory', get_option( 'gutenberg-experiments' ) ) : false,
+		'__experimentalEnableFullSiteEditing'   => $experiments_exist ? array_key_exists( 'gutenberg-full-site-editing', get_option( 'gutenberg-experiments' ) ) : false,
```

Comment:
There's a few more places where we have to update things because of this change.

See: https://github.com/WordPress/gutenberg/pull/17207/commits/2aa1d041c93bc7bf691bbc7dfc193c466cbb1a40